src/makepad.rs

```diff
@@ -58,40 +85,23 @@ pub(crate) fn copy_makepad_resources<P>(dist_resources_dir: P) -> std::io::Resul
 where
     P: AsRef<Path>
 {
-    let makepad_widgets_resources_dest = dist_resources_dir.as_ref()
-        .join("makepad_widgets")
-        .join("resources");
-    let makepad_widgets_resources_src = {
-        let cargo_metadata = MetadataCommand::new()
-            .exec()
-            .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, e))?;
-
-        let makepad_widgets_package_res = cargo_metadata
-            .packages
-            .iter()
-            .find(|package| package.name == "makepad-widgets")
-            .ok_or_else(|| std::io::Error::new(
-                std::io::ErrorKind::NotFound,
-                "makepad-widgets package not found"
-            ));
-
-        let _ = IS_MAKEPAD_APP.set(makepad_widgets_package_res.is_ok());
+    let makepad_resources_paths = get_makepad_resources_paths();
+    if makepad_resources_paths.is_empty() {
+        // This situation can happen if the user use local Makepad repository and deletes the all `makepad-*.path` files.
+        return Err(std::io::Error::new(std::io::ErrorKind::NotFound, "No Makepad resources found"));
```

Comment:
great job catching this scenario! 

Let's be a bit more detailed here, so they know what to look for and how to fix it.

```suggestion
        return Err(std::io::Error::new(
            std::io::ErrorKind::NotFound,
            "Missing resource paths: no `.path` files found in the Makepad build directory (./target/release/)",
        ));
```